golang/x/relay/genesis.go

```diff
@@ -52,10 +52,16 @@ func DefaultGenesisState() GenesisState {
 }
 
 // InitGenesis inits the app state based on the genesis state
-func InitGenesis(ctx sdk.Context, keeper Keeper, data GenesisState) []abci.ValidatorUpdate {
-	keeper.SetGenesisState(ctx, data.Headers[0], data.PeriodStart)
-	keeper.IngestHeaderChain(ctx, data.Headers[1:])
-	return []abci.ValidatorUpdate{}
+func InitGenesis(ctx sdk.Context, keeper Keeper, data GenesisState) ([]abci.ValidatorUpdate, sdk.Error) {
+	err := keeper.SetGenesisState(ctx, data.Headers[0], data.PeriodStart)
+	if err != nil {
```

Comment:
go ahead and revert this, it's changed in my branch

Reply:
Ok, reverted